src/metric.ts

```diff
@@ -109,29 +112,101 @@ export class NetScore extends Metric {
   }
 }
 
+
```

Comment:
This should be a function in api_access.ts. I'm aiming to make all API calls in api_access.ts, keeping them out of metric.ts. Metric.ts is just for calculating the actual metrics. When you move it to api_access.ts, see if you can change it from a class to function.